osu.Game/Screens/Play/HUD/LegacyComboCounter.cs

```diff
@@ -120,20 +118,41 @@ private set
         }
 
         [BackgroundDependencyLoader]
-        private void load(ScoreProcessor scoreProcessor)
+        private void load(ScoreProcessor scoreProcessor, ISkinSource skin)
         {
             Current.BindTo(scoreProcessor.Combo);
+
+            // Since layout depends on combo font height we need to update it during skin change
+            skin.SourceChanged += updateLayout;
```

Comment:
Event is never unsubscribed from. Compare [example](https://github.com/ppy/osu/blob/9ceceeca3d038ac2917a0dd56fc83e8459d08ffd/osu.Game.Rulesets.Catch/UI/CatcherTrailDisplay.cs#L131-L132).